src/api/tests/queryMocks.ts

```diff
@@ -5,24 +5,33 @@ import { BusinessContact, ContactType } from '../entities/BusinessContact';
 import { Participant } from '../entities/Participant';
 import { User } from '../entities/User';
 
-export const mockParticipant = (participant: Partial<Participant> | null = {}) => {
-  jest.spyOn(Participant, 'query').mockReturnValueOnce(
-    QueryBuilder.forClass(Participant).resolve(
-      participant === null
-        ? undefined
-        : {
-            id: 1,
-            name: 'Test Participant',
-            location: 'Test Location',
-            allowSharing: true,
-            types: [
-              {
-                id: 1,
-                typeName: 'DSP',
-              },
-            ],
-            ...participant,
-          }
+type PartialParticipantOrNull = Partial<Participant> | null;
+export const mockParticipant = (
+  participant: PartialParticipantOrNull | PartialParticipantOrNull[] = {}
+) => {
+  const participants = Array.isArray(participant) ? participant : [participant];
```

Comment:
This makes me wonder if this should just be two different methods, `mockParticipant` and `mockParticipants`. Does that make any sense or no?

Reply:
This is the pattern we use for the rest of the functions in this file.

Reply:
fair, ignore me :D